src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC } from 'react';
+import { TodoTask } from '../TodoTask';
+import { Todo } from '../../types/Todo';
+import { ErrorType } from '../../types/Error';
+
+type Props = {
+  filteredTodos: Todo[];
+  tempTodo: Todo | null;
+  processing: number[];
+  error: ErrorType;
```

Comment:
The 'processing' prop should have a more descriptive name, like 'processingTodoIds'.